pkg/kedge/http/director/proxy.go

```diff
@@ -158,10 +167,15 @@ func (t *backendPoolTripper) RoundTrip(req *http.Request) (*http.Response, error
 	return tripper.RoundTrip(req)
 }
 
-func respondWithError(err error, req *http.Request, resp http.ResponseWriter) {
+func (p *Proxy) respondWithError(err error, req *http.Request, resp http.ResponseWriter) {
 	errType := errtypes.RouteUnknownError
 	if err == router.ErrRouteNotFound {
 		errType = errtypes.NoRoute
+
+		// Add additional information if user is trying to connect to a specific IP address.
```

Comment:
Can we catch that on winch as well?

Reply:
I moved this logic to winch, I guess we don't really need it in kedge, or do we?

Reply:
It would be nice to have it both TBH

Reply:
But it's ok to start with winch